kong/cmd/start.lua

```diff
@@ -25,6 +25,9 @@ end
 local lapp = [[
 Usage: kong start [OPTIONS]
 
+Start Kong (Nginx and other configured services) in configured
```

Comment:
rather say "Nginx and other dependencies" or something along that line ... "services" sounds too broad, indeterminate.